build.gradle

```diff
@@ -82,8 +82,11 @@ repositories {
 dependencies {
 
   compile group: 'cascading', name: 'cascading-hadoop2-mr1', version: cascadingVersion, changing: true
+  compile group: 'org.apache.commons', name: 'commons-csv', version: '1.2'
+  
+  compile group: 'com.twitter.elephantbird', name: 'elephant-bird-core', version: '4.13'
```

Comment:
You can copy the file and leave the copyright intact. We do that in Cascading-Hbase for instance. The Apache License allows this.